test/models/config_test.rb

```diff
@@ -39,20 +53,22 @@ class ConfigTest < ActiveSupport::TestCase
     end
 
     it 'should validate URLs' do
-      Config::CONFIG_URLS.each do |url_field|
-        c = Config.find_or_create_by(config_key: url_field)
-        
-        # confirm cleanup cleanup
-        c.config_value = { options: ["www.efax.com/path"] }
-        assert c.valid?
-
-        # bad
-        c.config_value = { options: ["bad url"] }
-        refute c.valid?
-
-        # allow no URL
-        c.config_value = { options: [] }
-        assert c.valid?
+      Config::CLEAN_PRE_VALIDATION
+        .select{ |field, cleaner| cleaner == :clean_url }
+        .each do |url_field, cleaner|
```

Comment:
Also not running! I think you actually want:

```suggestion
      Config::VALIDATIONS
        .select{ |field, validator| validator == :validate_url }
        .each do |url_field, _validator|
```

(the `_` on `_validator` is us acknowleding we aren't using this variable!)